qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -38,10 +38,15 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	InvalidVirtualMethod(
 			"The virtual method `{0}` in the type `{1}` is not applicable for the base object `{2}` type."), //
 	InfixNotationParameterRequired("A parameter for the infix notation method `{0}` is required."), //
-	InvalidMethodInfixNotation("The method `{0}` cannot be used with infix notation, because it has not `1` parameter."), //
-	
+	InvalidMethodInfixNotation(
+			"The method `{0}` cannot be used with infix notation, because it has not `1` parameter."), //
+	MethodNotSupportedInNativeMode("Method `{0}` of `{1}` Java type cannot be used in native images mode."), //
+	SuperMethodNotSupportedInNativeMode(

```

Comment:
I'm thinking maybe to call this "Inherited method" instead of "Super method". I know the option is called ignoreSuperClasses, but usually when you are using "super", the fields/methods you have access to are referred to as inherited.

Reply:
I agree, fixed.